tests/s3/versioning/test_delete_object_tagging.py

```diff
@@ -53,8 +55,8 @@ def setup_method(self):
         self.s3_test_obj = S3TestLib(endpoint_url=S3_CFG["s3_url"])
         self.s3_ver_obj = S3VersioningTestLib(endpoint_url=S3_CFG["s3_url"])
         self.s3_tag_obj = S3TaggingTestLib(endpoint_url=S3_CFG["s3_url"])
-        self.ver_tag = dict()
-        self.versions = dict()
+        self.ver_tag = {}
```

Comment:
That was correct only. It does not give any codacy warning in files

Reply:
Yes, But pylint gives following warning.
Consider using {} instead of dict()  (use-dict-literal)

and even if literal are available then no need to use wrapper. literals precedes wrappers.